lib/excoveralls/task/util.ex

```diff
@@ -54,6 +54,17 @@ Usage: mix coveralls.post <Options>
     -s (--sha)          Commit SHA (required when not using Travis)
     --build             Service number ('BUILDS' column at coveralls.io page)
     --parallel          coveralls.io 'parallel' option (See coveralls.io API Reference)
+    --flagname         coveralls.io 'flag_name' option (See coveralls.io API Reference)
```

Comment:
same thing here, this is the `.post` section.

Reply:
https://github.com/duffelhq/excoveralls/pull/1/commits/9282870949692b965b8c2ae3e9102755256f43bf 

Here as well